src/gsy_e/models/strategy/scm/load.py

```diff
@@ -0,0 +1,63 @@
+from typing import Dict, TYPE_CHECKING
+
+from pendulum import DateTime
+
+from gsy_e.models.strategy.load_hours import LoadHoursPerDayEnergyParameters
+from gsy_e.models.strategy.predefined_load import DefinedLoadEnergyParameters
+from gsy_e.models.strategy.scm import SCMStrategy
+
+if TYPE_CHECKING:
+    from gsy_e.models.area import AreaBase
+
+
+class SCMLoadHoursStrategy(SCMStrategy):
+    """Load SCM strategy with constant power production."""
+    def __init__(self, avg_power_W, hrs_per_day=None, hrs_of_day=None):
+        self._energy_params = LoadHoursPerDayEnergyParameters(avg_power_W, hrs_per_day, hrs_of_day)
+        self._simulation_start_timestamp = None
+
+    def activate(self, area: "AreaBase") -> None:
+        """Activate the strategy."""
+        self._energy_params.event_activate_energy(area)
+        self._simulation_start_timestamp = area.now
+
+    def _get_day_of_timestamp(self, time_slot: DateTime) -> int:
+        """Return the number of days passed from the simulation start date to the time slot."""
+        if self._simulation_start_timestamp is None:
+            return 0
+        return (time_slot - self._simulation_start_timestamp).days
+
+    def market_cycle(self, area: "AreaBase") -> None:
+        """Update the load forecast and measurements for the next/previous market slot."""
+        # self._update_energy_requirement_in_state()
+        # # Provide energy values for the past market slot, to be used in the settlement market
+        # self._set_energy_measurement_of_last_market()
```

Comment:
Just only saw this. So you are overwriting the `market_cycle` method and do nothing here, right? Please remove the commented code then. thanks!

Reply:
Fixed